services/scan/src/precinct_scanner_state_machine.ts

```diff
@@ -483,14 +494,22 @@ function buildMachine(createPlustekClient: CreatePlustekClient) {
         accepted: {
           entry: storeAcceptedSheet,
           invoke: pollPaperStatus,
-          on: {
-            SCANNER_NO_PAPER: { target: 'accepted', internal: true },
-            SCANNER_READY_TO_SCAN: {
-              target: 'ready_to_scan',
-              actions: clearLastScan,
+          initial: 'scanning_paused',
+          on: { SCANNER_NO_PAPER: { target: undefined } }, // Do nothing
+          states: {
+            scanning_paused: {
+              on: { SCANNER_READY_TO_SCAN: { target: undefined } }, // Do nothing
+              after: {
+                DELAY_ACCEPTED_READY_FOR_NEXT_BALLOT: 'ready_for_next_ballot',
+              },
             },
+            ready_for_next_ballot: {
+              on: { SCANNER_READY_TO_SCAN: '..ready_to_scan' },
+            },
+          },
+          after: {
+            DELAY_ACCEPTED_RESET_TO_NO_PAPER: 'no_paper',
```

Comment:
I'd been trying up until this point to avoid using nested states, but I think it's the clearest way to implement this situation with the two delays.

Curious how it is reading it - there are other places where nested states could also be used (e.g. when we use a series of states to complete one operation and check that it succeeded - e.g. rejecting)

Reply:
This reads pretty clean to me!

Reply:
Glad to hear it!